### PR TITLE
Bump phpstan to level 3

### PR DESCRIPTION
**Pull request by jdrieghe**
### Fixed
- We now reached PHPStan Level 3

---
Damn, this PR seriously ran out of hand :/

### CODE REVIEW

File: src/EventExport/Format/TabularData/TabularDataEventFormatter.php [hunk 1]
```diff
@@ -767,12 +768,9 @@ private function parseEventIdFromUrl($event): string
      * Gives a formatter that tries to fetch a summary in plain text.
      * If the formatted summary is missing, the summary that is available on the event will be used as fallback.
      *
-     * @param Format                                  $format
-     * @param CalendarSummaryRepositoryInterface|null $calendarSummaryRepository
-     *
-     * @return string
+     * @return Closure
```
Comment (bertramakers) [inline src/EventExport/Format/TabularData/TabularDataEventFormatter.php:771]: I think the `Closure` class is usable in 7.1 so maybe we should just add that on the method itself like you did for the `CalendarSummaryRepositoryInterface` argument? 

https://www.php.net/manual/en/class.closure.php
> (PHP 5 >= 5.3.0, PHP 7, PHP 8)

File: src/Media/Serialization/MediaObjectSerializer.php [hunk 2]
```diff
@@ -4,20 +4,17 @@
 
 use CultuurNet\UDB3\Iri\IriGeneratorInterface;
 use CultuurNet\UDB3\Media\Image;
-use CultuurNet\UDB3\Media\MediaObject;
 use CultuurNet\UDB3\Media\Properties\MIMEType;
 use Symfony\Component\Serializer\Exception\UnsupportedException;
-use Symfony\Component\Serializer\SerializerInterface;
 
-class MediaObjectSerializer implements SerializerInterface
+class MediaObjectSerializer
```
Comment (bertramakers) [inline src/Media/Serialization/MediaObjectSerializer.php:10]: Good call 👍

**Review by bertramakers — APPROVED** [review]
Nice cleanup! 👏 
It was "een hele boterham" to go through but the small commits made it bearable 😄 
Just some minor questions.

File: src/Event/ReadModel/JSONLD/EventLDProjector.php [hunk 3]
```diff
@@ -525,31 +521,28 @@ function (array $jsonLd) use ($ownerChanged) {
...
+            return [];
         }
 
         try {
             $placeJSONLD = $this->placeService->getEntity(
                 $placeId
             );
 
-            return json_decode($placeJSONLD);
+            return (array) json_decode($placeJSONLD);
```
[the remark this answers is not in the document]
Comment (jdrieghe) [reply src/Event/ReadModel/JSONLD/EventLDProjector.php:536]: I tried to keep existing behaviour here, I'm not sure of the implications.

[the remark this answers is not in the document]
Comment (bertramakers) [reply src/Event/ReadModel/JSONLD/EventLDProjector.php:536]: Ah you're right, I didn't think of it that way. Makes sense that if it casted to `(array)` in the usages that we do the same for now here 👍

**Comment by bertramakers** [comment]
Extra commits look good to me! 👍

**Pull request merged**